manifests/init.pp

```diff
@@ -130,7 +114,7 @@
 class simp (
   Enum['simp','simp_lite','poss'] $scenario                   = 'simp',
   Boolean                         $enable_data_includes       = true,
-  String                          $data_includes_array_name   = 'classes',
+  Optional[Array]                 $classes                    = lookup('simp::classes', Optional[Array], { 'strategy' => 'deep', 'knockout_prefix' => '--' }, undef),
```

Comment:
@lamawithonel Yeah, this needs to be pushed down into the data layer so that it can be overridden later.